### ISSUE TITLE
How to make stacked bar chart

### ISSUE THREAD

**Issue by whruiqlik** [AnyChart/AnyChart-Qlik#31]
How to use AnyChart to make stacked bar chart? Thank you.

![QlikSense](https://user-images.githubusercontent.com/43289168/71080824-e1947480-21c8-11ea-9e29-88df185e26b6.png)
![Data](https://user-images.githubusercontent.com/43289168/71080838-e5c09200-21c8-11ea-875c-a259cbebb34a.png)

**Comment by whruiqlik** [comment]
Can I make a stacked bar chart without changing the data structure?
Because we need to do bar charts and pie charts besides stacked bar chart. 
Thank you.

**Comment by Shestac92** [comment]
@whruiqlik 
Does the table in the Qlik Sense look exactly the same as you provided in your previous comment at the screenshot?

**Comment by whruiqlik** [comment]
After changing the data structure, I don't know how to make the following bar chart and pie chart.

![20191219175150](https://user-images.githubusercontent.com/43289168/71164223-94c4a280-2289-11ea-9ac1-b6e35f09c3f4.png)

**Comment by whruiqlik** [comment]
Can I make a stacked bar chart without changing the data structure?  Thank you.

![20191231180849](https://user-images.githubusercontent.com/43289168/71618007-b2cec300-2bf8-11ea-858a-f08c32e4f980.png)

**Comment by Shestac92** [comment]
@whruiqlik 
Yes, it's possible to build the same chart without changing the data structure. 
![Screenshot 2020-01-09 at 11 14 37](https://user-images.githubusercontent.com/24609062/72035126-f220a200-32d1-11ea-8c26-a7bbb0cb97c2.png)

The main trick is to apply the specific function expression to measures to sum the `amount` by `material`.

The following FX are required `Sum ( {$<[material]={'a'}>} amount)` and `Sum ( {$<[material]={'b'}>} amount)`.
![Screenshot 2020-01-09 at 11 14 57](https://user-images.githubusercontent.com/24609062/72035258-4f1c5800-32d2-11ea-90fe-916fe9eb51ab.png)

The file below is the QVF file with the chart on the screenshots above.
[whruiqlik_sample.qvf.zip](https://github.com/AnyChart/AnyChart-Qlik/files/4038429/whruiqlik_sample.qvf.zip)

**Comment by whruiqlik** [comment]
Thank you very much.
